deployment/localstack/terraform/main.tf

```diff
@@ -1,4 +1,10 @@
 module "aws" {
   source              = "../../../iac/terraform/aws"
   environment         = "localstack"
+  bloodconnect_domain = "example.com"
+
+  providers = {
+    aws.us-east-1 = aws.us-east-1
+  }
+  depends_on = [ aws_acm_certificate.name, aws_route53_zone.main ]
```

Comment:
i believe we have security check error on this

Reply:
It's only on localstack. Deploying on AWS without any issue. Created a separated ticket regarding this problem